rules/REQUEST-903.9001-DRUPAL-EXCLUSION-RULES.conf

```diff
@@ -310,4 +310,52 @@ SecAction "id:9001380,\
 	ctl:ruleRemoveTargetById=942450;REQUEST_COOKIES_NAMES,\
 	ctl:ruleRemoveTargetById=942450;REQUEST_COOKIES" 
 
+
+#
+# [ WYSIWYG/CKEditor assets and upload ]
+# 
+# Disable the unnecessary requestBodyAccess and for binary uploads
+#
+# 
+#
+SecRule REQUEST_METHOD "^POST$" \
+        "id:'9001390',\
+        phase:1,\
+        t:none,\
+        pass,\
+        nolog,\
+        noauditlog,\
+        chain"
+                SecRule REQUEST_FILENAME "/admin/content/assets/add/[a-z]+$" chain
```

Comment:
Would you mind always setting the operator explicitly? Using @ rx here really makes sense.